arch/risc-v/src/rv64gc/riscv_assert.c

```diff
@@ -132,63 +132,171 @@ static inline void riscv_registerdump(volatile uintptr_t *regs)
 }
 
 /****************************************************************************
- * Name: up_taskdump
+ * Name: riscv_dump_stak
```

Comment:
```suggestion
 * Name: riscv_dump_task
```
Typo in the function name.